tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+requires =
+    tox >= 4
+env_list = py38, build-seerep, integration-examples
+skipdist = True
+
+[testenv]
+basepython = python3.8
+description = run tests
+allowlist_externals = bash
+
+[testenv:build-seerep]
+description = build seerep
+commands = bash -c "source ../devel/setup.bash;\
+                    catkin build"
+
+[testenv:{integration-examples,light}]
```

Comment:
What's the purpose of having `light` has a second name for the environment configuration?

Reply:
> What's the purpose of having `light` has a second name for the environment configuration?

I thought of using it as an alias, to run just that environment. But after rethinking using the more descriptive name is probably better and avoids possible confusion. I now removed the `light` environment.